io/plugins/eu.esdihumboldt.hale.io.gml.geometry/src/eu/esdihumboldt/hale/io/gml/geometry/handler/ArcHandler.java

```diff
@@ -78,20 +81,23 @@ public Object createGeometry(Instance instance, int srsDimension, IOProvider rea
 				.getParameter(InterpolationConstant.INTERPOL_MAX_POSITION_ERROR).as(Double.class);
 
 		if (maxPositionalError == null || maxPositionalError.doubleValue() <= 0) {
-			log.warn(
-					"Value of Max positional error parameter, for interpolation operation, is not valid. Default value has been taken.");
+			if (reportedWarning.compareAndSet(false, true)) {
```

Comment:
This is not ideal, as it will only give a warning once in a running instance of hale, and not once per read operation. But it is definitely an improvement over the log spamming.